### ISSUE TITLE
Auto Generate Whitelist Data Install Script

### ISSUE THREAD

**Issue by mpchadwick** [rhoerr/supee-6788-toolbox#3]
Rather than just doing this https://github.com/rhoerr/supee-6788-toolbox/blob/186ecb2cada4913c9e8850635c15a6bb780518ce/fixSUPEE6788.php#L705 it might be nice if this created a data install script for the necessary blocks and variables (perhaps optionally). That way you can deploy the upgrade script rather than running this in production. Magento "technical details" also recommend a data install script. I'm sure everyone is busy getting this patch out, but if anyone has time (I can _maybe_, but probably not PR this myself).

**Comment by rhoerr** [comment]
The tech details recommendation was in regards to affected modules, and in that context it does make sense. Creating a module, you'd want to make sure any blocks needed are added to the whitelist when it gets installed.

To do that, though, you need a module to attach the data upgrade to. Don't have that in this context. You could generate everything needed, for sure, but that's getting rather involved.

**Comment by mpchadwick** [comment]
Yeah, it certainly wouldn't be trivial, but would be nice...even just to get the install script only would save _some_ time and then it would be on the developer to create the module to make sure it gets executed...

**Comment by chelevich** [comment]
created a PR to get the upgrade scripts from log file
